index.html

```diff
@@ -578,8 +578,8 @@ <h1 class="section-title wow fadeInUp" data-wow-delay="0.2s">Got Questions? Here
         <div id="Con-Us" class="row" >
           
             <ul id="ContactList" class="wow fadeInDown" data-wow-delay="0.3s">
-              <li class="contact-li"><i class="lni-map-marker c-icon"></i>Chandigarh Patiala National Highway</li>
-              <li  class="contact-li"> &emsp;&emsp; Rajpura, Punjab - 140401</li>
+              <li class="contact-li"><i class="lni-map-marker c-icon"></i>Chandigarh-Patiala National Highway </li>
+              <li  class="contact-li"><i class="lni-map-marker c-icon" style="opacity: 0;"></i>Rajpura, Punjab - 140401</li>

```

Comment:
Is the alignment fixed due to icon taking space before the text or is there some left padding? (can't look through the css since it's minified)